src/components/organisms/table/alphabet-filter/alphabet-filter.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+import { useEffect } from "react";
+import { TableConstants } from "@constants";
+import { AlphabetLetter } from "@molecules";
+import { Li, Ul, Section } from "@atoms";
+
+interface AlphabetLetterProps {
+  activeLetter: string;
+  availableLetters: Array<string>;
+  getAllAlphabetLettersFunction: Function;
+  alphabetFilterFunction: Function;
+}
+
+export const AlphabetFilter: React.FC<AlphabetLetterProps> = ({
+  activeLetter,
+  availableLetters,
+  getAllAlphabetLettersFunction,
+  alphabetFilterFunction,
+}) => {
+  useEffect(() => {
+    getAllAlphabetLettersFunction();
+  }, []);
+
+  const onCharSelect = (char: string) => {
+    alphabetFilterFunction(char);
+  };
+
+  return (
+    <Section aria-label="Alphabet Filter">
+      <Ul role="list" className="application-content__alphabet">
+        {TableConstants.alphabetList.map((char) => (
+          <Li key={char}>
+            <AlphabetLetter
+              activeLetter={activeLetter}
+              alphabet={availableLetters}
+              char={char}
+              onCharSelect={onCharSelect}
```

Comment:
Why aren't we calling `alphabetFilterFunction` directly here? Also I would exclude the `Function` part of the name and rename it to onFilter or something similar